src/test/resources/Test/TestGCS1.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Demo1
+
+  @TC-Demo-
+  Scenario: User is able to Login and confirm data is getting transferred from GCS to BigQuery
+    Given Open Datafusion Project to configure pipeline1
+    When Source is GCS bucket1
+    When Target is BigQuery1
+    Then Link Source and Sink to establish connection1
```

Comment:
Then should contain assertions. Any actions should go into "when". And any preconditions ("source is", "target is") go to "given"